Header header
int64 num_of_neighbor
string neighbors
float32[] pos_xs
float32[] pos_ys
float32[] pos_zs
float32[] co2s
float32[] temperature_reading
float32[] humidity_reading



